omnibus/package-scripts/agent-deb/prerm

```diff
@@ -158,6 +158,9 @@ remove_apm_remote_config()
     fi
 }
 
+if [ -f "$INSTALL_DIR/embedded/bin/python" ]; then  
+    ${INSTALL_DIR}/embedded/bin/python "${INSTALL_DIR}/python-scripts/prerm.py" "${INSTALL_DIR}" || true
```

Comment:
Do we need to run this when we're uninstalling? It seems only relevant when upgrading no?

Reply:
Good catch! Fixed!